reports/engagement_ratio/MauDau.ipynb

```diff
@@ -159,6 +205,52 @@
     "    diff = generated - target\n",
     "    return diff.days <= 10\n",
     "\n",
+    "\n",
+    "from moztelemetry.standards import filter_date_range, count_distinct_clientids\n",
+    "\n",
+    "# Similar to the version in standards.py, but uses subsession_start_date\n",
```

Comment:
Yes, ideally we'd make the "mau" and "dau" functions in `standards.py` a bit more flexible so they could be used on dataframes with arbitrary column names and formats. This was the quick and dirty method.

I've filed [an issue](https://github.com/mozilla/python_moztelemetry/issues/64) in the python_moztelemetry repo to follow up with a better approach.